vendors/espressif/boards/esp32/ports/ble/bluedroid/iot_ble_hal_gatt_server.c

```diff
@@ -791,6 +795,24 @@ BTStatus_t prvBTSendResponse( uint16_t usConnId,
 
 /*-----------------------------------------------------------*/
 
+static BTStatus_t prvBTConfigureMtu( uint8_t ucServerIf,
```

Comment:
ucServerIf is not used. Is it expected?

Reply:
Yes that's expected. Bluedroid API doesn't need the GATT server interface to set the local MTU so parameter is unused for the port.